Resources/Computing/ComputingElement.py

```diff
@@ -472,7 +472,7 @@ def getResourceDict( ceName = None ):
   # now add some defaults
   resourceDict['Setup'] = gConfig.getValue( '/DIRAC/Setup', 'None' )
   if not 'CPUTime' in resourceDict:
-    resourceDict['CPUTime'] = maxCPUSegments[-1]
+    resourceDict['CPUTime'] = gConfig.getValue( '/LocalSite/CPUTime' )
```

Comment:
Shoud the first value be the default one ? 
resourceDict['CPUTime'] = gConfig.getValue( '/LocalSite/CPUTime',  maxCPUSegments[-1])

The CE can be used not only in the local site context